packages/xstate-vue/src/useActorRef.ts

```diff
@@ -35,12 +35,14 @@ export function useActorRef<TLogic extends AnyActorLogic>(
   const actorRef = createActor(actorLogic, options);
 
   let sub: Subscription;
-  onMounted(() => {
-    if (observerOrListener) {
-      sub = actorRef.subscribe(toObserver(observerOrListener));
-    }
+
+  if (observerOrListener) {
+    sub = actorRef.subscribe(toObserver(observerOrListener));
+  }
+
+  if (typeof window !== 'undefined' && typeof document !== 'undefined') {
     actorRef.start();
```

Comment:
Now the code still won't quite work exactly the same bewteen client and the server. In the thread you have described that the current inner workings suffer from:
> It makes the actor unusable during SSR;


And now... they will still work the same way on the server. So it seems one of the things you have wanted to address with this PR just can't be addressed.